gradle-plugins/compose/src/test/test-projects/misc/commonResources/expected-open-res/androidMainResourceCollectors/my/lib/res/ActualResourceCollectors.kt

```diff
@@ -0,0 +1,53 @@
+@file:OptIn(org.jetbrains.compose.resources.InternalResourceApi::class)
+
+package my.lib.res
+
+import kotlin.OptIn
+import kotlin.String
+import kotlin.collections.Map
+import org.jetbrains.compose.resources.DrawableResource
+import org.jetbrains.compose.resources.ExperimentalResourceApi
+import org.jetbrains.compose.resources.FontResource
+import org.jetbrains.compose.resources.PluralStringResource
+import org.jetbrains.compose.resources.StringArrayResource
+import org.jetbrains.compose.resources.StringResource
+
+@ExperimentalResourceApi
```

Comment:
Let's create a task of removing `@ExperimentalResourceApi` in 1.8 or 1.9 (what is more appropriate in your opinion).